qiskit/primitives/backend_estimator.py

```diff
@@ -67,14 +68,17 @@ def _run_circuits(
         max_circuits = backend.max_circuits
     else:
         raise RuntimeError("Backend version not supported")
-    if max_circuits:
-        jobs = [
-            backend.run(circuits[pos : pos + max_circuits], **run_options)
-            for pos in range(0, len(circuits), max_circuits)
-        ]
-        result = [x.result() for x in jobs]
-    else:
-        result = [backend.run(circuits, **run_options).result()]
+    with warnings.catch_warnings():
+        # TODO remove this catch once Aer stops using QobjDictField
+        warnings.simplefilter("ignore", category=DeprecationWarning)
```

Comment:
In these cases where the warning is coming from a module outside of qiskit (or you need some more granularity because you might be suppressing unwanted warnings), it might make sense to filter warnings by module, something like:

```suggestion
        warnings.filterwarnings("ignore", category=DeprecationWarning, module=r"qiskit_aer(\.[a-zA-Z0-9_]+)*")
```

it's something I have tried to do in past deprecations, for example: [here](https://github.com/Qiskit/qiskit/pull/10792/files#diff-7bcb16fad17dea49a12edcf727cdf3c32ef85fec8c97f5cb3b14bdca155611bfR117).

For an even better control, you can also specify a concrete message to filter with, like it's done in the [QiskitTestCase](https://github.com/Qiskit/qiskit/blob/419f40e9e72b4981a6536d08d6b0fa91c13021d5/test/utils/base.py#L234C1-L239C10) class.

Reply:
Excelente punto. Agregado en c2f2fdb3de97a6f5a44c8a6ba73442e75f99b106